package.json

```diff
@@ -34,7 +34,7 @@
     "test": "cd app && yarn test",
     "pre-commit-lint:app": "cd app && yarn pre-commit-lint",
     "pre-commit-lint": "yarn pre-commit-lint:app",
-    "typecheck": "cd app && yarn typecheck"
```

Comment:
why removing typecheck? we can switch to yarn workspace syntax though (no need to `cd`)

Reply:
my bad, on my main machine it fails locally so I had it temporarily disabled, must have accidentally committed the change